ql/CMakeLists.txt

```diff
@@ -965,7 +960,6 @@ set(QuantLib_HDR
     config.msvc.hpp
```

Comment:
I added these few `config.*.hpp` files back — they are unused by cmake, but having them here made it a lot easier to restore the filelist check.  This way, there's an automated check that the list is kept up to date when someone adds new files to the library.